app/controllers/admin_v2/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+# rubocop:disable Rails/ApplicationController
```

Comment:
We may be able to inherit from ApplicationController a little later, for now its safer and simpler to keep this isolated.